.circleci/config.yml

```diff
@@ -251,7 +251,7 @@ jobs:
   integration-test-for-jdbc-postgresql:
     docker:
       - image: circleci/openjdk:8-jdk
-      - image: circleci/postgres:9.6.9-alpine
+      - image: postgres:9.6.24-alpine
```

Comment:
Can you please upgrade to PostgreSQL 12. It's the latest version we support:
https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md

Reply:
Just a note. I benchmarked Scalar DL on PostgreSQL 14 (the latest stable) with TPCC and nothing went wrong.
So, maybe it's good enough to say 14. Of course, we can (or should) wait for the QA team verification.

Reply:
@thongdk8 So let's use PostgreSQL 14.

Reply:
I updated the version of the databases. Please take a look again. Thank you.